src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/appsettings.json

```diff
@@ -4,7 +4,7 @@
     "SblAuthCookieName": ".ASPXAUTH",
     "JwtCookieName": "AltinnStudioRuntime",
     "BaseUrl": "http://localhost",
-    "BridgeAuthnApiEndpoint": "https://at22.altinn.cloud/sblbridge/authentication/api/",
+    "BridgeAuthnApiEndpoint": "https://httpstat.us/503",
```

Comment:
was this commited intentionally?

Reply:
No, good catch. Restoring original value.